<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_einstellung_dxf</class>
 <widget class="QDialog" name="Dialog_einstellung_dxf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grundeinstellungen DXF</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>381</width>
     <height>126</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_KennungUnterseite"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Kennung Datei für Oberseite</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_dezimaltrenner">
      <property name="text">
       <string>Dezimalzeichen in Klassennamen</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Trennzeichen von Parametern im Klassennamen</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_dezimaltrenner"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_parametertrenner"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_KennungOberseite"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Kennung Datei für Unterseite</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Unterseite ist gedreht um</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="comboBox_drehdypUnterseite"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_abbrechen">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_parametertrenner</tabstop>
  <tabstop>lineEdit_dezimaltrenner</tabstop>
  <tabstop>lineEdit_KennungOberseite</tabstop>
  <tabstop>lineEdit_KennungUnterseite</tabstop>
  <tabstop>comboBox_drehdypUnterseite</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_abbrechen</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
